frontend/packages/console-shared/src/components/formik-fields/text-column-field/TextColumnField.tsx

```diff
@@ -21,7 +20,7 @@ const TextColumnField: React.FC<TextColumnFieldProps> = (props) => {
       name={name}
       render={(arrayHelpers) => (
         <>
-          <DndProvider backend={HTML5Backend}>
+          <DragAndDrop>
```

Comment:
@invincibleJai Request you to take a look at this DnD implementation change.

Reply:
Fixed the `error: cannot-have-two-html5-backends-at-the-same-time` issue, which was observed in other areas like git Import -> healthChecks and containerSourceForm. Now iI am using the useRef to save the HTML5Backend to reuse the backend between the renders to avoid the issue.